Modules/IO/DCMTK/src/itkDCMTKImageIO.cxx

```diff
@@ -31,6 +31,26 @@
 #include "dcmtk/dcmdata/dcrledrg.h"
 #include "dcmtk/oflog/oflog.h"
 
+namespace
+{
+
+/**
+ * Helper function to test for 128 byte dicom preamble
+ * @param file A stream to test if the file is dicom like
+ * @return true if the stream has a dicom preamble
+ */
+static bool
+readPreambleDicom(std::ifstream & file) // NOTE: This file is duplicated in itkGDCMImageIO.cxx
+{
+  unsigned char preamble[132];
+
+  file.read(reinterpret_cast<char *>(preamble), 132);
```

Comment:
Please consider, instead:

```cpp
    char preamble[132] = "";

    file.read(preamble, sizeof(preample));
```
- properly initializing `preamble` to avoid undefined behavior in the `return` statement (especially in the case when `file.read` has silently failed).
- avoiding unnecessary `reinterpret_cast<char *>`
- avoiding typing the same magic number `132` twice.